Source/WTF/wtf/Assertions.cpp

```diff
@@ -193,9 +193,9 @@ static void vprintf_stderr_with_prefix(const char* prefix, const char* format, v
     memcpy(formatWithPrefix.data() + prefixLength, format, formatLength);
     formatWithPrefix[prefixLength + formatLength] = 0;
 
-    ALLOW_NONLITERAL_FORMAT_BEGIN
+ALLOW_NONLITERAL_FORMAT_BEGIN
```

Comment:
I guess this is opposite of what @grorg commented but every instance of `ALLOW_NONLITERAL_FORMAT_END` in WebKit currently is indented

I don't really care either way however

Reply:
Before this patch, "\nALLOW_" has 420 hits in WebCore, whereas "    ALLOW_" has 258 hits. so I think Dean is actually right - we seem to be preferring the unindented form

Reply:
Across the whole WebKit workspace, though, "\nALLOW_" has 833 hits, whereas " ALLOW_" has 1125.

So I guess it's not super clear what I should do. Maybe I should just leave the indentation alone?

Reply:
After discussing offline, we're comfortable merging this patch as-is.